src/messaging/ReliableMessageManager.cpp

```diff
@@ -492,9 +491,8 @@ void ReliableMessageManager::ClearRetransmitTable(RetransTableEntry & rEntry)
         rEntry.rc->Release();
         rEntry.rc = nullptr;
 
-        if (rEntry.msgBuf)
+        if (!rEntry.msgBuf.IsNull())
```

Comment:
Could just make the `rEntry.msgBuf = nullptr` unconditional.  I don't think there's a good reason to do the null-check here and another one inside the `operator=`.

Reply:
Done. By the way, thanks for taking the time to review thoroughly; with so many similar changes I definitely get a bit mechanical.

Reply:
I totally understand that.  ;)  Thank you again for doing all this work!